modules/fbs/fbs.module

```diff
@@ -99,10 +99,17 @@ function fbs_ding_entity_is($object, $class) {
       // display and has a chance of calling opensearch to get marc record. We
       // need none of this and only want to know if the material is reservable
       // and if it has any periodical information.
-      $result = fbs_service()->Catalog->getHoldings(
-        fbs_service()->agencyId,
-        $object->localId
-      );
+      try {
+        $result = fbs_service()->Catalog->getHoldings(
+          fbs_service()->agencyId,
+          $object->localId
+        );
+      }
+      catch(Exception $e) {
+	watchdog_exception('fbs', $e);
+        return FALSE;
```

Comment:
Should we return `NULL` here since we actually cannot tell anything about the reservability of a material when FBS returns an error?

[`is()` will return `FALSE` if no hook returns `TRUE`](https://github.com/ding2/ding2/blob/c0273fbf051044e9ac6971cad104df70cc4c7786/modules/ding_entity/ding_entity.module#L372-L388).